model/clusters_mgmt/v1/wif_config_status_type.model

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright (c) 2024 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Configuration status of a WifConfig.
+class WifConfigStatus {
```

Comment:
WifConfigStatus has been defined as a class. In this metalanguage the class keyword creates a structure that includes an id, kind, and href field. For this type of object, I don't believe these fields are necessary. The 'struct' keyword should be used instead. This will create the data structure without these implicit fields.
```suggestion
struct WifConfigStatus {
```

Reply:
Done